config/routes.rb

```diff
@@ -1,8 +1,11 @@
 Rails.application.routes.draw do
+  resources :comments
   resources :media
   resources :authorships
   mount_devise_token_auth_for 'User', at: 'auth'
-  resources :users
+  resources :users do
+    resources :comments
+  end
   resources :sections do
     resources :articles do
       resources :media
```

Comment:
Triple nested route haysus crysto. Remove the media route underneath articles.

Reply:
Remind me to talk about this in tomorrow's meeting. We need to actually think about this and remove all unnecessary routes.